README.md

```diff
@@ -149,8 +146,7 @@ Use a command such as this to import the keys:
 $ gpg --keyserver pool.sks-keyservers.net --recv-keys DD8F2338BAE7501E3DD5AC78C273792F7D83545D
 ```
 
-_(See the bottom of this README for a full script to import active
-release keys)_
+(See the bottom of this README for a full script to import active release keys.)
```

Comment:
Maybe remove the parentheses as well?

Reply:
@richardlau Yes, that's a good suggestion. Done!